server/services/emailService.js

```diff
@@ -0,0 +1,30 @@
+import nodemailer from 'nodemailer';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const transporter = nodemailer.createTransport({
+  service: 'Gmail',
```

Comment:
We should have these on heroku

Reply:
@seyi-adeleke yes, Victor will add those.